x-pack/plugin/security/src/main/java/org/elasticsearch/xpack/security/enrollment/tool/BootstrapPasswordAndEnrollmentTokenForInitialNode.java

```diff
@@ -1,144 +0,0 @@
-/*
- * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
- * or more contributor license agreements. Licensed under the Elastic License
- * 2.0; you may not use this file except in compliance with the Elastic License
- * 2.0.
- */
-
-package org.elasticsearch.xpack.security.enrollment.tool;
-
-import joptsimple.OptionSet;
-import joptsimple.OptionSpec;
-
-import org.elasticsearch.cli.ExitCodes;
-import org.elasticsearch.cli.KeyStoreAwareCommand;
-import org.elasticsearch.cli.Terminal;
-import org.elasticsearch.cli.UserException;
-import org.elasticsearch.common.Strings;
-import org.elasticsearch.common.settings.KeyStoreWrapper;
-import org.elasticsearch.common.settings.SecureString;
-import org.elasticsearch.common.settings.Settings;
-import org.elasticsearch.common.xcontent.XContentBuilder;
-import org.elasticsearch.common.xcontent.json.JsonXContent;
-import org.elasticsearch.core.CheckedFunction;
-import org.elasticsearch.env.Environment;
-import org.elasticsearch.xpack.core.security.user.ElasticUser;
-import org.elasticsearch.xpack.security.authc.esnative.ReservedRealm;
-import org.elasticsearch.xpack.core.security.EnrollmentToken;
-import org.elasticsearch.xpack.security.enrollment.EnrollmentTokenGenerator;
-import org.elasticsearch.xpack.core.security.CommandLineHttpClient;
-import org.elasticsearch.xpack.core.security.HttpResponse;
-
-import java.io.IOException;
-import java.net.HttpURLConnection;
-import java.net.MalformedURLException;
-import java.net.URISyntaxException;
-import java.net.URL;
-import java.util.function.Function;
-
-import static org.elasticsearch.xpack.core.security.CommandLineHttpClient.createURL;
-import static org.elasticsearch.xpack.security.tool.CommandUtils.generatePassword;
-
-public class BootstrapPasswordAndEnrollmentTokenForInitialNode extends KeyStoreAwareCommand {
```

Comment:
Since the design change to not spawn off an external process on startup to generate the password and token, this is not needed any more